pkg/signal/signal_unix.go

```diff
@@ -8,5 +8,9 @@ import (
 
 // Signals used in api/client (no windows equivalent, use
 // invalid signals so they don't get handled)
+
+// signal sent to a process when a child process terminates, is interrupted, or resumes after being interrupted.
 const SIGCHLD = syscall.SIGCHLD
```

Comment:
It's used in ``api/client/start.go`` to forward signals. I think it's to be able to use this signal on all platform (see https://github.com/docker/docker/blob/master/pkg/signal/signal_windows.go#L11) without doing a *condition*.